src/features/delegation/hooks/useDelegatees.ts

```diff
@@ -63,27 +64,46 @@ async function fetchDelegateeStats(
     ),
   });
 
+  const delegatedFractionResults = await publicClient.multicall({
+    contracts: metadata.map(
+      (d) =>
+        ({
+          address: Addresses.LockedGold,
+          abi: lockedGoldABI,
+          functionName: 'getAccountTotalDelegatedFraction',
+          args: [d.address],
+        }) as const,
+    ),
+  });
+
   // Process validator lists to create list of validator groups
   const delegatees: AddressTo<Delegatee> = {};
   for (let i = 0; i < metadata.length; i++) {
     const address = metadata[i].address as Address;
 
     const lockedBalanceRes = lockedBalanceResults[i];
     const votingPowerRes = votingPowerResults[i];
+    const delegatedFractionRes = delegatedFractionResults[i];
 
-    if (lockedBalanceRes.status !== 'success' || votingPowerRes.status !== 'success')
+    if (
+      lockedBalanceRes.status !== 'success' ||
+      votingPowerRes.status !== 'success' ||
+      delegatedFractionRes.status !== 'success'
+    )
       throw new Error('Error fetching delegatee stats');
 
     const lockedBalance = lockedBalanceRes.result as bigint;
     const votingPower = votingPowerRes.result as bigint;
-    const delegatedBalance = votingPower - lockedBalance;
+    const votingPowerMinusLockedBalance = votingPower - lockedBalance;
+    const delegatedByPercent = fromFixidity(delegatedFractionRes.result as bigint) * 100;
 
     delegatees[address] = {
       ...metadata[i],
       address,
       lockedBalance,
       votingPower,
-      delegatedBalance,
+      delegatedToBalance: votingPowerMinusLockedBalance > 0 ? votingPowerMinusLockedBalance : 0n,
```

Comment:
if we have the amount they are delegating then the true delated amount would be 

votingPower - (lockedBalance - (lockedBalance * delegatedByPercent))

right?

this way we dont need to show the incorrect 0 when its negative

Reply:
used 

```solidity  
// Celo that was delegated to this particular address
mapping(address => uint256) public totalDelegatedCelo;
```

directly instead as discussed